sympy/functions/special/delta_functions.py

```diff
@@ -528,6 +548,12 @@ class is about to be instantiated and it returns either some simplified
         >>> Heaviside(x - 100).subs(x, 105)
         1
 
+        Parameters
+        ==========
+
+        arg : argument passed by heaveside object
+        HO : boolean flag for HeaviSide Object is set to True
```

Comment:
I guess it should be `Heaviside` (in both places)

Reply:
Done.